packages/hawtio/src/plugins/camel/CamelContent.tsx

```diff
@@ -28,6 +28,7 @@ import { Trace } from './trace'
 import * as ccs from './camel-content-service'
 import { CamelRoutes } from '@hawtiosrc/plugins/camel/routes/CamelRoutes'
 import { Source } from '@hawtiosrc/plugins/camel/routes/Source'
+import { SendMessage } from '@hawtiosrc/plugins/camel/endpoints/SendMessage'
```

Comment:
Since it's a component within the same plugin, we should use relative path instead of absolute starting from `@hawtiosrc/`. Same for the above Camel components.